tests/CMakeLists.txt

```diff
@@ -1,5 +1,8 @@
 find_package(Boost 1.55.0 REQUIRED)
 
+# We move the test files into the testing directory
+file(COPY "${CMAKE_CURRENT_SOURCE_DIR}/hypervolume_test_data/" DESTINATION "${CMAKE_BINARY_DIR}/tests/hypervolume_test_data/")
```

Comment:
but then the tests are not finding the test files, marcus expects them to be in:
```
/ root directory of the hypervolume data
    std::string input_data_dir("./hypervolume_test_data/");
```